mteb/evaluation/evaluators/Image/ClusteringEvaluator.py

```diff
@@ -70,7 +68,6 @@ def __call__(self, model: Encoder, *, encode_kwargs: dict[str, Any] = {}):
 
         return {
             "v_measure": v_measure,
-            "accuracy": accuracy,
```

Comment:
> changed image clustering main metric to nmi & delete accuracy

Any reason to go for nmi over the v-measure?

Reply:
They actually always produce the same values, while NMI is a more known metric in image clustering I think! (e.g., [benchmark](https://paperswithcode.com/sota/image-clustering-on-imagenet) [benchmark](https://paperswithcode.com/sota/image-clustering-on-tiny-imagenet) [paper](https://openaccess.thecvf.com/content_ICCV_2017/papers/Chang_Deep_Adaptive_Image_ICCV_2017_paper.pdf) [paper](https://arxiv.org/pdf/2402.10093v2) [paper](https://arxiv.org/pdf/2303.17896v2))

Reply:
Thanks! Then there is no worries